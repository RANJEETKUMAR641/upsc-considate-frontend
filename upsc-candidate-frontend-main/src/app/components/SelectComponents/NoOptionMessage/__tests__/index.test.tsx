import * as React from 'react'
import { render } from '@testing-library/react'

import { NoOptionMessage } from '..'

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: (str) => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    }
  },
}))

describe('<NoOptionMessage  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<NoOptionMessage />)
    expect(loadingIndicator.container.firstChild).toMatchSnapshot()
  })
})
