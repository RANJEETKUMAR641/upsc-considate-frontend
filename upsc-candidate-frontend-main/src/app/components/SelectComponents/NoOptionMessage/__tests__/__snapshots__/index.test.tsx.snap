// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoOptionMessage  /> should match snapshot 1`] = `
<div>
  dmeo
</div>
`;
